Justfile

```diff
@@ -32,8 +32,31 @@ format-generated-code:
   #!/usr/bin/env -S nix shell --accept-flake-config 'github:freckle/flakes?dir=main#fourmolu-default' --command bash
   fourmolu -i recurly-client/src --unsafe
 
-# Use version.txt as the version in the .cabal file
-set-package-version:
-  #!/usr/bin/env bash
+patch-cabal-file:
+  #!/usr/bin/env -S nix shell 'nixpkgs#haskellPackages.cabal-fmt' --command bash
+
+  # Use version.txt as the version in the .cabal file
```

Comment:
Why can't this project use a `package.yaml` and our usual release patterns?

Reply:
The code generator produces the Cabal file and I don't think it can produce an Hpack file.

Reply:
Would it be a Bad Idea to just maintain our own `package.yaml` and tell the code generator not to generate any `.cabal`? Or, worst case, do something like ` <generate> && rm *.cabal && stack build --dry-run`?

It seems to me, even for a generate-to-hackage situation like this, you'd still want pretty tight control over the package details itself. I think the hacky stuff you've got here is evidence of that too. Figuring out a way to just author the `package.yaml` / cabal file yourself seems like it'd be more robust over time.